extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzBuildTimeConfig.java

```diff
@@ -41,6 +41,15 @@
     @ConfigItem(name = "datasource")
     public Optional<String> dataSourceName;
 
+    /**
+     * The prefix for quartz job store tables.
+     * <p>
+     * Optional - defaults to `QRTZ_` if not specified.
+     * Ignored if using a `ram` store.
+     */
+    @ConfigItem
```

Comment:
Minor but you should use `defaultValue = "QRTZ_"` in this annotation.

This has several advantages: you can drop the doc about the default value above and it being optional: it will be automatically documented. You can also simplify the configuration code below.

Reply:
Pushed changes - thank you for the review!